man/setkey.Rd

```diff
@@ -87,6 +87,10 @@ required.)
 If you really wish to use column numbers, it is possible but
 deliberately a little harder; e.g., \code{setkeyv(DT,names(DT)[1:2])}.
 
+If you use integer columns as keys, it's crucial to ensure correct behavior in subsetting and joining operations.
+Integer keys should be used with the dot (\code{.}) syntax to explicitly match values against the key 
+rather than interpreting them as indices. See examples for better understanding.
```

Comment:
"indices" here means row number and that is different from the function `indices`
so can you please change indices to row numbers here and in the comments below?